CHANGELOG.md

```diff
@@ -2,6 +2,91 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 55 end 2017-02-27
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+55+Ending+Feb+27%2C+2017%22+label%3Aenhancement)
+
+Automate
+  - Ansible
+    - Update the service dialog to use the correct automate entry point [(#13955)](https://github.com/ManageIQ/manageiq/pull/13955)
```

Comment:
Move this one and `Change default provisioning entry point for AutomationManagement` to the "Changed" section.